packages/editor/src/components/post-publish-button/index.js

```diff
@@ -66,17 +78,38 @@ export class PostPublishButton extends Component {
 			onSave();
 		};
 
+		const buttonProps = {
+			'aria-disabled': isButtonDisabled,
+			className: 'editor-post-publish-button',
+			isBusy: isSaving && isPublished,
+			isLarge: true,
+			isPrimary: true,
+			onClick,
+		};
+
+		const toggleProps = {
+			'aria-disabled': isToggleDisabled,
+			'aria-expanded': isOpen,
+			className: 'editor-post-publish-panel__toggle',
+			isBusy: isSaving && isPublished,
+			isPrimary: true,
+			onClick: onToggle,
+		};
+
+		const toggleChildren = isBeingScheduled ? __( 'Schedule…' ) : __( 'Publish…' );
+		const buttonChildren = <PublishButtonLabel forceIsSaving={ forceIsSaving } />;
+
+		const componentProps = isToggle ? toggleProps : buttonProps;
+		const componentChildren = isToggle ? toggleChildren : buttonChildren;
 		return (
 			<Button
 				ref={ this.buttonNode }
-				className="editor-post-publish-button"
-				isPrimary
-				isLarge
-				onClick={ onClick }
-				disabled={ isButtonDisabled }
```

Comment:
For what reason did we need to stop passing the `disabled` prop? While this and many other buttons in the header _appear_ as disabled, they don't act as disabled. You can immediately press "Publish" on a new post and the publish panel is (wrongly?) shown.

As it relates to focus return, it's not clear to me how it's relevant, since the panel would have presumably never been opened if it were disabled.

Reply:
Good catch. https://github.com/WordPress/gutenberg/pull/13194 should fix it.

The `disabled` property didn't play well with the `withFocusReturn` HOC, IIRC. I can dig up the exact details if necessary.